.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Run linter
```

Comment:
Also, you can integrate golangci lint with your IDE to show errors from linters directly when you're writing code. I'm not sure how to do it in VSCode (but it's probably possible), but in Neovim you can wrap CLI tools(linters) in the LSP server using [none ls](https://github.com/nvimtools/none-ls.nvim). 
<img width="1599" alt="image" src="https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/assets/93580374/309f4051-2401-4ce7-a4df-666e74c3163c">

Reply:
I have made it my default configuration for go linting in vscode, so I still get linting errors

Reply:
yeah, but I cloned your repo - and my linter doesn't work. Configuration in the repo will make it work for me also :)
Not all guys have golangci configured as a default linter

Reply:
Also, consider following case:
1. Programmer #1 has golangcilint configured with specific default rules in his IDE
2. Programmer #2 has also golangcilint configured as a default linter for his IDE, but with different rules.

Both guys will get different errors from their's linters, because they have different configuration for them. Golangci lint configuration in the repo will unify/standardize rules, and both programmers will receive same errors -> code will be more consistent :)

Reply:
@hrvadl thank for pointing this out, fixed!